### ISSUE TITLE
Structure pattern: icons on each row lack title or aria-label

### ISSUE THREAD

**Issue by erral** [plone/mockup#965]
The action buttons present on each line of the structure pattern table don't have have any title or aria-label marker or any text explaining their purpose

![2020-04-02 14-27-04-ko-pantaila-argazkia](https://user-images.githubusercontent.com/817365/78249091-0bf70d00-74ee-11ea-8623-5acbf493fa1c.png)

Other buttons such the action buttons (cut, paste, delete, ...) have an aria-label attribute.

![2020-04-02 14-29-35-ko-pantaila-argazkia](https://user-images.githubusercontent.com/817365/78249327-74de8500-74ee-11ea-8c10-3d0cfbd7c0a0.png)

@codesyntax/plone 

@ionlizarazu
